qutebrowser/misc/sql.py

```diff
@@ -469,9 +533,19 @@ def select(self, sort_by, sort_order, limit=-1):
 
         Return: A prepared and executed select query.
         """
-        q = Query("SELECT * FROM {table} ORDER BY {sort_by} {sort_order} "
-                  "LIMIT :limit"
-                  .format(table=self._name, sort_by=sort_by,
-                          sort_order=sort_order))
+        q = self._database.query(
+            f"SELECT * FROM {self._name} ORDER BY {sort_by} {sort_order} LIMIT :limit"
+        )
         q.run(limit=limit)
         return q
+
+
+def version() -> str:
+    """Return the sqlite version string."""
+    try:
+        in_memory_db = Database(':memory:')
+        version = in_memory_db.query("select sqlite_version()").run().value()
```

Comment:
Clashes with the `version` function it is in - I typically use shorter names for smaller scopes, i.e. something like `ver` here (or if you prefer, `sqlite_version` for the function).

Reply:
I no longer need to store the result in a variable with that `contextlib.closing` trick :)